x-pack/plugins/actions/index.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Root } from 'joi';
+import mappings from './mappings.json';
+import { init } from './server';
+
+export { ActionsPlugin, ActionsClient } from './server';
+
+export function actions(kibana: any) {
+  return new kibana.Plugin({
+    id: 'actions',
+    configPrefix: 'xpack.actions',
+    require: ['kibana', 'elasticsearch', 'encrypted_saved_objects'],
+    config(Joi: Root) {
+      return Joi.object()
+        .keys({
+          enabled: Joi.boolean().default(true),
+        })
+        .default();
+    },
+    init,
+    uiExports: {
+      mappings,
```

Comment:
Using a single "saved object type" for all actions means we won't be able to restrict users to only access certain "action types", similar to the discussion which I raised here: https://github.com/elastic/kibana/issues/36836

If we're going to want to restrict users to a subset of actions in the future, it might be advantageous to create different saved object types for the various alert types.

Reply:
Similar to what I wrote above, I'm thinking we do a future phase post design discussion to properly implement feature controls / actions limiting.